index.js

```diff
@@ -250,7 +250,10 @@ export function useGesture(props, config) {
       handleGesture('onDrag')
     }
 
-    const cancelDrag = event => requestAnimationFrame(() => onDragEnd(event, true))
+    const cancelDrag = event => {
+      updateState({ drag: { canceled: true } })
```

Comment:
`updateState` is a commodity to clarify relatively complex state updates, but since it's a very straightforward update to state, I think `state.current.drag.canceled = true` should work!